app/api/products/schema.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+from pydantic import BaseModel
+
+
+class BaseSchema(BaseModel):
+    class Config:
+        extra = "forbid"
+        allow_population_by_field_name = True
+
+
+class Response(BaseSchema):
+    message: str
+    status: str
+
+
+class ProductDataParams(BaseModel):
+    station_ids: List[int] = [67774010, 67963040]
+    period: List[str] = ['2000-01-01', '2000-03-01']
+    elements: List[int] = [2, 4]
```

Comment:
To allow easier testing this is the default values that will be populated when testing in the interactive documentation

Reply:
OK, I understand that these are just the defaults and other data params can be passed to the `products/climatic-summary` (or any other product endpoint).